src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -54,271 +40,59 @@ class DashboardListingUi extends React.Component {
-                defaultMessage="You have {totalDashboards} dashboards, but your {listingLimitText} setting prevents
-                the table below from displaying more than {listingLimitValue}. You can change this setting under {advancedSettingsLink}."
-                values={{
-                  totalDashboards: this.state.totalDashboards,
-                  listingLimitValue: this.props.listingLimit,
-                  listingLimitText: (
-                    <strong>
-                      listingLimit
-                    </strong>
-                  ),
-                  advancedSettingsLink: (
-                    <EuiLink href="#/management/kibana/settings">
-                      <FormattedMessage
-                        id="kbn.dashboard.listing.listingLimitExceeded.advancedSettingsLinkText"
-                        defaultMessage="Advanced Settings"
-                      />
-                    </EuiLink>
-                  )
-                }}
-              />
-            </p>
-          </EuiCallOut>
-          <EuiSpacer size="m" />
-        </React.Fragment>
-      );
-    }
   }
 
-  renderNoResultsMessage() {
-    if (this.state.isFetchingItems) {
-      return '';
-    }
-
+  render() {
+    const { intl } = this.props;
     return (
-      <FormattedMessage
-        id="kbn.dashboard.listing.noMatchedDashboardsMessage"
-        defaultMessage="No dashboards matched your search."
+      <TableListView
+        create={this.props.create}
+        find={this.props.find}
+        delete={this.props.delete}
+        edit={this.props.edit}
+        tableColumns={this.getTableColumns()}
+        listingLimit={this.props.listingLimit}
+        initialFilter={this.props.initialFilter}
+        hideWriteControls={this.props.hideWriteControls}
+        noItemsFragment={this.getNoItemsMessage()}
+        entityName={
+          intl.formatMessage({
+            id: 'kbn.dashboard.listing.table.entityName',
+            defaultMessage: 'dashboard',
+          })
+        }
+        entityNamePlural={
+          intl.formatMessage({
+            id: 'kbn.dashboard.listing.table.entityNamePlural',
+            defaultMessage: 'dashboards',
+          })
+        }
+        tableListTitle={
+          intl.formatMessage({
+            id: 'kbn.dashboard.listing.dashboardsTitle',
+            defaultMessage: 'Dashboards',
+          })
+        }
       />
     );
   }
 
-  renderNoItemsMessage() {
-
+  getNoItemsMessage() {
     if (this.props.hideWriteControls) {
       return (
-        <EuiText>
-          <h2>
-            <EuiTextColor color="subdued">
-              <FormattedMessage
-                id="kbn.dashboard.listing.noDashboardsItemsMessage"
-                defaultMessage="Looks like you don't have any dashboards."
-              />
-            </EuiTextColor>
-          </h2>
-        </EuiText>
+        <div>
+          <EuiEmptyPrompt
+            iconType="visualizeApp"
+            title={
+              <h2>
```

Comment:
Should this be a `h1`? Looks like the most important heading on the page
<img width="715" alt="screenshot 2019-02-18 at 11 17 21" src="https://user-images.githubusercontent.com/1508364/52944037-c7062a80-336e-11e9-9e37-afaa216cb65e.png">